### ISSUE TITLE
Communications

### ISSUE THREAD

**Issue by maisutton** [hyphacoop/distributed-press-organizing#107]
Comms Pipeline

- [x] PIECE: Announcement of #102 and #103 releases
- [x] PIECE: Announcement of #104 release
- [ ] PIECE: Relating M3 ^^ releases with Federation/Decentralization angle
- [ ] PIECE: Federation and Threads integration post
- [ ] PIECE: DWeb publishing + Fediverse for activists - alternatives for digital sovereignty
- [ ] Record videodemo with the slides and the script we wrote for Fediforum - 📹 

Social Media Posts

- [ ] Projects that implemented DP? (human rights org) - Im Defensoras
- [ ] Fediverse/threads updates
- [x] Primer for non-Fediverse + general audience / COMPOST readers could understand
- [ ] Sutty post: https://sutty.nl/en/how-to-publish-your-sutty-posts-in-fediverse/
- [ ] Start planning: How to explain moderation on Sutty -- TEASER + Look over UX story
- [x] IPFS Ecosystem Directory Listing: https://airtable.com/appLWiIrg9SQaEtEq/shrjwvk9pAeAk0Ci7

Websites

- [ ] Update news.compost.digital

Comms Channels

- [x] DP site - blog 
- [x] Fediverse: AP Fediverse publishing for DP blog
- [x] Twitter: @dwebpress
- [x] Bluesky: @dwebpress (2024dwebpress) 
- [?] Instagram: @COMPOSTmag 
- Matrix: https://matrix.to/#/#distributed-press-collab:tomesh.net
- Open Collective: https://opencollective.com/distributed-press

- Newswire? https://github.com/hyphacoop/distributed-press-organizing/issues/107

**Comment by maisutton** [comment]
* Communication goals
    * Continue to prioritize communicating benefits of dweb publishing to activists
    * Explain why it's helpful and what the point is
    * Fediverse integration needs more publicity
        * What would they like to see, what would they add?
        * Use as a small discovery activity
    * On social media, there was resonance with the fact that there are other spaces besides X/Instagram -- comms to boost other spaces, point to alternatives
    * Ania's story: Textbook for high schoolers, referenced news pieces/Youtube videos, URLs were breaking down and inaccessible, so client asked to download everything and upload to Google drive
        * Problem with using URLs and their failures/impermanence
* Do we have distributed.press solutions to address link rot more directly?
        * QR codes? IPFS links?
    * Mai needs to understand which features we will be rolling out and can be communicated. But we need to have the productization clearer in order to improve this strategy.
    * Customers? news agencies, education.
    * New funding opportunities: how can we productize in a way where some are willing to pay / others will find it useful even if unable to pay
        * Apply to more grants to bolster our infrastructure, alongside research/awareness of direction of productization and future income

**Comment by sutty-coop** [comment]
In our reverse timeline, we estimated:
|   -7 | Product offering page is published (Feb 2nd) 

@maisutton in case you want to add this to the Pipeline, and/or change the deadline

**Comment by sutty-coop** [comment]
@maisutton regarding the Distributed Press Offerings, we have added comments and edited a few lines in this document you sent:
https://hackmd.io/GL376L6FRGKc0IHkSuzNfw?edit
I've also found another similar document when reviewing issues:
https://hackmd.io/QMBk2PELQ5ONIJta3bAJSw?edit
Don't know if you planned to take a bit from both or else, just noticed that and letting you know just in case!

**Comment by akhileshthite** [comment]
@maisutton As per the TODO, I've simplified the language and changed the order. Also, made some comments.
https://hackmd.io/GL376L6FRGKc0IHkSuzNfw?both=

**Comment by sutty-coop** [comment]
Ania is drafting the PIECES: 
Announcement of #101  https://github.com/hyphacoop/distributed-press-organizing/issues/102 https://github.com/hyphacoop/distributed-press-organizing/issues/103 releases

**Comment by sutty-coop** [comment]
Texts available here:

102 https://hackmd.io/8SfCQUT5T0iYPIqtr-4-FQ?view -- EN ready with images ✔ WIP translating to Spanish 🛑 
101 https://hackmd.io/ScgfyUX5SFm5KOpz65nEWw?view -- EN ready with images ✔ WIP translation to Spanish 🛑 
104 https://hackmd.io/q8HOqiCZTz2BH-9VsST78Q  -- translated to Spanish ✔ WIP adding images 

We'll wait until the specs is polished 👇 
103 https://hackmd.io/4_5xZFnpTYivsU7WC6JFsA?view 

These are announcement of releases. Simple and straightforward pieces, for people in the “TL:DR” vibe. There’s a short sentence in the beginning of each announcement, emphasizing on the mission of DP, incorporating by Ben, Akhi, Mauve when talking about the X factor, etc. We could improve it. 
Then, the announcement.
Finally, a section when we repeat “🕸️What you can already do with Distributed Press:” in case they are coming across this text for the first time and don’t knot the rest of the project. I know it’s repetitive, but it could be a strategy.

**Comment by sutty-coop** [comment]
- [x] Footer for text pieces
- [x] Hyping the Fediverse account of Distributed.Press during March
     - [x] following folks and orgs
     - [x] sharing previous texts and documentation

**Comment by sutty-coop** [comment]
Graphic pieces produced for this milestone (thanks to Librenauta from Sutty for the original images!)


![Image](https://github.com/hyphacoop/distributed-press-organizing/assets/113475467/c33f7480-bcd3-42b4-b2de-452a7730f0a8)
![Image](https://github.com/hyphacoop/distributed-press-organizing/assets/113475467/7c3bd17b-beb5-46e9-b6ab-367ab24c44d7)
![Image](https://github.com/hyphacoop/distributed-press-organizing/assets/113475467/69ed47a7-8427-4329-9c88-4f7a6234373d)
![Image](https://github.com/hyphacoop/distributed-press-organizing/assets/113475467/3b799d3c-8f21-458f-81c1-9258a4fa2e9d)
![Image](https://github.com/hyphacoop/distributed-press-organizing/assets/113475467/98775720-d383-43b1-a1af-86db55cc348c)
![Image](https://github.com/hyphacoop/distributed-press-organizing/assets/113475467/a32be879-18c6-48eb-beef-22b165b13513)
![Image](https://github.com/hyphacoop/distributed-press-organizing/assets/113475467/2cefb3f7-3450-47cc-bdc5-2b95b9940d75)
![Image](https://github.com/hyphacoop/distributed-press-organizing/assets/113475467/c5324838-b41d-46ed-887d-15cef0646e55)


![Image](https://github.com/hyphacoop/distributed-press-organizing/assets/113475467/e46b2891-76e3-4b5b-9536-43ae09d86edd)

**Comment by sutty-coop** [comment]
April 19th: Moderation queue release.

1. Post in DP blog + fediverse reflection
a) Turn on Fediverse publishing for DP blog (distributed.press/blog)
b) Post this piece in English https://hackmd.io/q8HOqiCZTz2BH-9VsST78Q
c) Retoot through Compost, Sutty, Hypha and personal accounts

2. Post in X @dwebpress
 a) Create a square sized set of images saying:

Social media is exhausting (and problematic!). But websites aren't enough for outreach.
We are creating tools that gather the best of both platforms, away from Big Tech.
Join Distributed.Press and use our free tools to share your content with alternative publishing models.



![Image](https://github.com/hyphacoop/distributed-press-organizing/assets/113475467/949cfed2-4904-489e-8a97-54dc241c43ad)
![Image](https://github.com/hyphacoop/distributed-press-organizing/assets/113475467/a5c7a734-cb0d-4818-a9cc-e22474daa123)
![Image](https://github.com/hyphacoop/distributed-press-organizing/assets/113475467/fe051372-63cc-482e-8e70-738b1c337f16)




Get distributed! 
Find our tools and newest releases here: https://distributed.press/

b) Then post a thread of tweets with the information of this piece: https://hackmd.io/q8HOqiCZTz2BH-9VsST78Q
c) Ask for reposts from Compost, Suttym Hypha, FFDW, etc

**Comment by writerly03** [comment]
Hey Ania - this looks like a good plan to me. I like the messaging for X; I think it's fine to post it there as a 'come follow us elsewhere' message, but obviously up to the rest of the team. Do we use Bluesky? Maybe another less awful but more mainstream option?

**Comment by sutty-coop** [comment]
> Hey Ania - this looks like a good plan to me. I like the messaging for X; I think it's fine to post it there as a 'come follow us elsewhere' message, but obviously up to the rest of the team. Do we use Bluesky? Maybe another less awful but more mainstream option?

Yes, there is a Bluesky account, but I think that we haven't had activity there yet, I'm not familiar with Bluesky yet but I can catch up. I'll ask Mai for the password and start replicating things there

**Comment by RangerMauve** [comment]
+1 to posting to bsy. There's folks on there that might be curious about our approach and TBH I would be happy to integrate with bsky once they actually open up federation outside their walled garden.

**Comment by RangerMauve** [comment]
Oh sorry misread, I'm not against posting on X but mayb e we could lock the thread so we don't need to worry about comments and just use it for retweets/showing up in timelines?

**Comment by AnaFukelman** [comment]
See #188 and #189

**Comment by AnaFukelman** [comment]
Post about DP CLI demo 
"Everything we do in 2.5 mins
This will go as YT short; it's 9:16 in ratio.
I need the YouTube embed link to add it to the docs.
Download it from here: https://bafybeierjaqi2u77oulmx4cxrxip46h2hjtfilw7mdznj3rsukgt2dto6u.ipfs.ipfs.hypha.coop/
Also, make a post on all our socials."

**Comment by AnaFukelman** [comment]
Closing this issue as it got too big, there are good ideas here to pick up in the future, will revisit it.
